ddtrace/contrib/openai/_endpoint_hooks.py

```diff
@@ -254,19 +255,20 @@ def _record_request(self, pin, integration, span, args, kwargs):
                 span.set_tag_str("openai.request.messages.%d.content" % idx, content)
                 span.set_tag_str("openai.request.messages.%d.role" % idx, role)
                 span.set_tag_str("openai.request.messages.%d.name" % idx, name)
+        return
+
+    def _record_response(self, pin, integration, span, args, kwargs, resp, error):
+        if not resp:
+            return self._handle_response(pin, span, integration, resp)
+        messages = kwargs.get("messages")
         if kwargs.get("stream"):
-            # streamed responses do not have a usage field, so we have to
-            # estimate the number of tokens returned.
             est_num_message_tokens = 0
+            estimated = False
```

Comment:
Estimated is a boolean here but it's then cast to an int. Dumb question. Is this intentional?

Reply:
`estimated` is just a placeholder span metric value that we use internally. For simplicity we cast it to an int when setting it as a span metric but in reality it's just 0(false) or 1 (true).